mpqp/qasm/open_qasm_2_and_3.py

```diff
@@ -290,7 +302,6 @@ def add_std_lib():
         )
         if new_instr is not None and new_instr not in included_instr:
             included_instr.add(new_instr)
-            header_code += qasm_code(new_instr)
```

Comment:
we removed this line because we are adding new_instr to included_instr, which means that the code writes the same instruction twice, one time in included_instr and header_code

Reply:
Ok makes sense, thanks